integration-libs/cpq-quote/cpq-quote-discount/components/cpq-quote-discount-tbody/cpq-quote.component.spec.ts

```diff
@@ -0,0 +1,151 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { CpqQuoteDiscountComponent } from './cpq-quote.component';
+import { CartItemContext, OrderEntry } from '@spartacus/cart/base/root';
+import { ReplaySubject, of } from 'rxjs';
+import { CpqDiscounts } from '../../../root/model';
+import { Component, Input } from '@angular/core';
+import { RouterTestingModule } from '@angular/router/testing';
+import { take } from 'rxjs/operators';
+import { CpqQuoteService } from '../../cpq-qute.service';
+
+class MockCartItemContext implements Partial<CartItemContext> {
+  item$ = new ReplaySubject<OrderEntry>(1);
+}
+
+@Component({
+  selector: 'cx-cpq-quote',
+  template: '',
+})
+class MockConfigureCpqDiscountsComponent {
+  @Input() cartEntry: Partial<OrderEntry & Array<CpqDiscounts>>;
+}
+
+describe('Cpq Quote Discount Component', () => {
```

Comment:
The name of the test should be `CpqQuoteDiscountComponent`.
Please, adjust it accordingly